### DISCUSSION TITLE
Is there a vue version??

### DISCUSSION THREAD

**Discussion by HarrySan001** [fkhadra/react-toastify#937] [Q&A] [answered]
I really appreciate `react-toastify`, and is there a vue version??

**Comment by jerrywu001** [comment, marked as answer]
Rewrite with vue3, called `vue3-toastify`

Docs:
https://vue3-toastify.js-bridge.com/

Npm:
https://www.npmjs.com/package/vue3-toastify

The same style, animation, and API is 85% the same as `react-toastify`.

Weekly Downloads > 4k

**Reply by HarrySan001** [reply]
Nice work! thanks~~
